conftest.py

```diff
@@ -1,12 +1,7 @@
-import os
 import pytest
 
-# Files to skip doctests in
-SKIP_DOCTESTS = ['dask/bytes/hdfs.py',
-                 'dask/array/fft.py']
-
-curdir = os.path.dirname(__file__)
-skip_doctests = {os.path.join(curdir, p) for p in SKIP_DOCTESTS}
+# The doctests in these files fail due to dependencies on travis
+collect_ignore = ['dask/bytes/hdfs.py', 'dask/array/fft.py']
```

Comment:
We should probably add a better comment for the `fft` part. In that case, `fft` has no doctests, but the functions it wraps from NumPy do, which is what pytest is detecting. However those tests fail when running, against their wrapped versions (e.g. they return Dask Arrays instead of NumPy arrays).